lib/killbill_client/models/transaction.rb

```diff
@@ -126,7 +130,7 @@ def follow_up_path(payment_id)
       def create_initial_transaction(path, query_map, payment_method_id, user, reason, comment, options)
         query_map[:paymentMethodId] = payment_method_id unless payment_method_id.nil?
 
-        with_payment_failure_handling do
+        created_transaction = with_payment_failure_handling do
```

Comment:
Since this is a bug fix, I'd prefer not adding new features. But we could definitely have a PR for this later.